src/Apps/Search/Routes/Artists/SearchResultsArtists.tsx

```diff
@@ -126,14 +126,14 @@ export class SearchResultsArtistsRoute extends React.Component<Props, State> {
 
   render() {
     const { viewer, location } = this.props
+    const { query } = location
     const { term } = get(location, l => l.query)
-
     const artists = get(viewer, v => v.search.edges, []).map(e => e.node)
     return (
       <LoadingArea isLoading={this.state.isLoading}>
         {artists.length === 0 ? (
           <Box mt={3}>
-            <ZeroState entity="artists" term={term} />
+            <ZeroState term={term} query={query} />
```

Comment:
Rather than passing stuff all around, could quickly do 
```tsx
<Subscribe to={[FilterState]}>
  {(filters: FilterState) => {
    const query = // parse filters state
    return <ZeroState term={term} query={query} />
   }}
</Subscribe>

Reply:
State is populated at the get go, [via `location.query`](https://github.com/damassi/reaction/blob/e60c0c35cba024e557fe80f391f3faa584317b40/src/Apps/Search/Routes/Artworks/SearchResultsArtworks.tsx#L22-L29).

Reply:
Ah, I see, got it! Will update.